docs/source/reference/jit-compilation.rst

```diff
@@ -405,6 +405,11 @@ C callbacks
 
       The address of the compiled C callback, as an integer.
 
+   .. attribute:: cffi
+
+      A `cffi`_ function pointer instance, suitable for passing as an
+      argument to `cffi`_-wrapped functions.
```

Comment:
We should note that the function pointer is `void *` so CFFI will not be able to verify pointer compatibility with the argument type.  Someone might be expected that kind of check to happen.